package.json

```diff
@@ -12,7 +12,8 @@
     "debug": "^2.1.1",
     "generic-pool": "^2.1.1",
     "underscore": "^1.7.0",
-    "underscore.deep": "^0.5.1"
+    "underscore.deep": "^0.5.1",
+    "coffee-script": "1.6.2"
```

Comment:
Could we use a newer version than 1.6?  We used 1.8 while developing this, and that should fix the test that is failing.